modules/logging-rn-5.7.7.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+// cluster-logging-release-notes.adoc
+// logging-5-7-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-7-7_{context}"]
+= Logging 5.7.7
+This release includes link:https://access.redhat.com/errata/RHBA-2023:5530[OpenShift Logging Bug Fix Release 5.7.7].
+
+[id="openshift-logging-5-7-7-bug-fixes_{context}"]
+== Bug fixes
+* Before this update, FluentD normalized the logs emitted by the EventRouter differently from Vector. With this update, vector produces log records in a consistent format. (link:https://issues.redhat.com/browse/LOG-4178[LOG-4178])
+
+* Before this update, there was an error in the query used for the *FluentD Buffer Availability* graph in the metrics dashboard created by the Cluster Logging Operator as it showed the minimum buffer usage. With this update, the graph shows the maximum buffer usage and is now renamed to *FluentD Buffer Usage*. (link:https://issues.redhat.com/browse/LOG-4555[LOG-4555])
+
+* Before this update, deploying LokiStack on IPv6-only or dual-stack OCP clusters caused the LokiStack memberlist registration to fail. As a result, the distributor pods went into a crash loop. With this update, the administrator can enable IPv6 using `lokistack.spec.hashRing.memberlist.enableIPv6: true` which resolves the issue. (link:https://issues.redhat.com/browse/LOG-4569[LOG-4569])
+
+* Before this update, the collector relied on the default configuration settings for reading the container log lines. As a result, the collector did not read the rotated files efficiently. With this update, there is an increase in the number of bytes read, which allows the collector to efficiently process rotated files. (link:https://issues.redhat.com/browse/LOG-4575[LOG-4575])
```

Comment:
```suggestion
* Before this update, the log collector relied on the default configuration settings for reading the container log lines. As a result, the log collector did not read the rotated files efficiently. With this update, there is an increase in the number of bytes read, which allows the log collector to efficiently process rotated files. (link:https://issues.redhat.com/browse/LOG-4575[LOG-4575])
```

Reply:
In the logging documentation, I see that this is commonly referred to as the log collector, not just the collector. This caught my eye because in Network Observability we have a flow collector

Reply:
Thanks form pointing this out. :) I have made the change. Thanks!